pwndbg/commands/spray.py

```diff
@@ -1,26 +1,49 @@
 import argparse
 
 import gdb
-from pwnlib.util.cyclic import cyclic_gen
+from pwnlib.util.cyclic import cyclic
 
+import pwndbg.color.message as M
 import pwndbg.commands
 
-parser = argparse.ArgumentParser(description="Spray memory with a cyclic_gen() generated values")
+parser = argparse.ArgumentParser(description="Spray memory with cyclic() generated values")
 parser.add_argument("addr", help="Address to spray")
-parser.add_argument("length", nargs="?", help="Length of byte sequence", type=int, default=0)
+parser.add_argument(
+    "length",
+    help="Length of byte sequence, when unspecified sprays until the end of vmmap which address belongs to",
+    type=int,
+    nargs="?",
+    default=0,
+)
+parser.add_argument("--value", help="Value to spray memory with", type=str, required=False)
```

Comment:
A 'value' to spray memory with sounds as if this is maybe an integer and e.g.:
- 0x41 -> 41 41 41 41 ...
- 0x4142 -> 41 42 41 42 ...
- 0x414243 -> 41 42 43 41 42 43 ... (likely not much usable)

Reply:
@gsingh93 wdyt?

Reply:
What if we say "hex byte string" instead, and the user would put `41`, `4142`, `414243`, etc? I think this would fix the confusion around endianness, and it would slightly simplify some code below if we don't need the `0x`.

Later we can also have a flag to accept a byte string instead of a hex byte string, so you could do "A", "AB", "ABC" instead.

Reply:
I think current implementation is fine, because you can put binary data and bytes without additional flag, we just need to tell user that hex strings are written in big endian order